lua/codesettings/build/schemas.lua

```diff
@@ -94,29 +95,49 @@ function M.get_schemas()
   return ret
 end
 
+--- A map of LSP server schemas that require special handling for non-common configuration layouts.
+local SpecialCases = {
+  nixd = function(json)
+    -- nixd should be nested under "nixd"
+    return { nixd = { type = 'object', properties = json.properties } }
+  end,
+  gopls = function(json)
+    local config = json.contributes.configuration.properties.gopls
+    json.description = config.markdownDescription
+
+    local properties = vim.empty_dict()
+    for k, v in pairs(config.properties) do
+      -- The official gopls documentation generally uses these options without prefixes and recommends doing so.
+      -- - https://github.com/golang/tools/blob/master/gopls/doc/editor/vim.md#neovim-config
+      -- - https://github.com/golang/tools/blob/master/gopls/doc/settings.md
+      local last = k:match("[^%.]+$")
+      properties["gopls." .. last] = v
```

Comment:
the quotes here are failing stylua checks, they should be single quotes

Reply:
fixed